### PR TITLE
Fix/snex feedback

### PR DESCRIPTION
**Pull request by dmcollom**
Added a small subset of the changes discussed in the response to Andy's document. Among those:

- Headers on pages indicating their purpose
- Hints
- Required email
- Changed some verbiage
- Reordered some import statements for reasons

### CODE REVIEW

File: tom_base/settings.py [hunk 1]
```diff
@@ -226,6 +226,9 @@
 
 THUMBNAIL_DEFAULT_SIZE = (200, 200)
 
+HINTS_ENABLED = False
+HINT_LEVEL = HINTS_ENABLED * 20
```
Comment (Fingel) [inline tom_base/settings.py]: Well, TIL that `True * 20 == 20` and `False * 20 == 0`. Clever, probably too clever. This is the kind of thing that breaks with a major Python release. 
Much simpler would be to simply write your own `add_message` function that checks settings.HINTS_ENABLED.

Comment (dmcollom) [reply tom_base/settings.py]: I actually spent quite a bit of time trying to override the MessageMiddleware to intercept and remove hint messages, but that proved to be more trouble than it was worth. This was the five minute solution. You're right, something in the middle is probably the better option...I'll update it.

Comment (Fingel) [reply tom_base/settings.py]: Can you do something like this?

```python
import messages
def add_hint(request, message, level=settings.HINT_LEVEL, *args, **kwargs):
    if settings.HINTS_ENABLED:
        messages.add_message(request, level, message, *args, **kwargs)
````

Comment (dmcollom) [reply tom_base/settings.py]: Yeah, that would have been the obvious, sane choice.

Comment (dmcollom) [reply tom_base/settings.py]: Fixed this, and added corresponding documentation for the setting.

**Pull request merged**